CHANGELOG.md

```diff
@@ -8,6 +8,7 @@ All notable changes to this project will be documented in this file.
 
 - Default stackableVersion to operator version ([#381]).
 - Configuration overrides for the JVM security properties, such as DNS caching ([#384]).
+- Support PodDisruptionBudgets ([#394]).
```

Comment:
Just to stay consistent, mention the operator-rs bump?

Reply:
Added. That's the reason I prefer to not mention to start with ^^